### PR TITLE
PMM-6746 Tests for PSMDB API.

### PR DESCRIPTION
**Pull request by BupycHuk**

### CODE REVIEW

File: management/dbaas/psmdb_cluster_test.go [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+
+//nolint:funlen
+func TestPSMDBClusterServer(t *testing.T) {
+	if pmmapitests.Kubeconfig == "" {
+		t.Skip("Skip tests of PSMDBClusterServer without kubeconfig")
+	}
+	registerKubernetesCluster(t, psmdbKubernetesClusterName, pmmapitests.Kubeconfig)
+
+	t.Run("BasicPSMDBCluster", func(t *testing.T) {
+		paramsFirstPXC := psmdbcluster.CreatePSMDBClusterParams{
```
Comment (percona-csalguero) [inline management/dbaas/psmdb_cluster_test.go:27]: Can we rename `paramsFirstPXC`? It is Mongo

File: management/dbaas/psmdb_cluster_test.go [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+			Context: pmmapitests.Context,
+			Body: psmdbcluster.CreatePSMDBClusterBody{
+				KubernetesClusterName: psmdbKubernetesClusterName,
+				Name:                  "first.pxc.test.percona.com",
```
Comment (percona-csalguero) [inline management/dbaas/psmdb_cluster_test.go:31]: not pxc

**Review by percona-csalguero — CHANGES_REQUESTED** [review]
Can we update all places saying PXC to PSMDB?

**Review by percona-csalguero — APPROVED** [review]

File: management/dbaas/psmdb_cluster_test.go [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+		paramsFirstPSMDB := psmdbcluster.CreatePSMDBClusterParams{
+			Context: pmmapitests.Context,
+			Body: psmdbcluster.CreatePSMDBClusterBody{
+				KubernetesClusterName: psmdbKubernetesClusterName,
+				Name:                  "first-psmdb-test",
+				Params: &psmdbcluster.CreatePSMDBClusterParamsBodyParams{
+					ClusterSize: 3,
+					Replicaset: &psmdbcluster.CreatePSMDBClusterParamsBodyParamsReplicaset{
+						ComputeResources: &psmdbcluster.CreatePSMDBClusterParamsBodyParamsReplicasetComputeResources{
+							CPUm:        1,
```
Comment (askomorokhov) [inline management/dbaas/psmdb_cluster_test.go:36]: Any idea what is the minimal requirements for CPUm?

**Review by askomorokhov — APPROVED** [review]
Looks good to me.

File: management/dbaas/psmdb_cluster_test.go (hunk 3 above)
Comment (BupycHuk) [reply management/dbaas/psmdb_cluster_test.go:36]: Unfortunatelly I don't know :(

**Pull request merged**